packages/client-api-schema/src/generated-schema.json

```diff
@@ -705,13 +705,6 @@
       ],
       "type": "object"
     },
-    "ErrorCode": {
-      "enum": [
-        200,
-        100
-      ],
-      "type": "number"
-    },
```

Comment:
Why is this here?

Reply:
I needed to run `yarn prepare` to update something (I think?), and it resulted in this change: https://github.com/statechannels/monorepo/pull/1491/commits/e47d753dae92d898c1a4585c215cdb4c3bcaec3a